tests/certification/state/mongodb/README.md

```diff
@@ -0,0 +1,24 @@
+# MongoDB State Store certification testing
+
+This project aims to test the MongoDB State Store component under various conditions.
+
+## Test plan
```

Comment:
See https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-mongodb/

Please add tests that verify the following metadata properties.

```
  - name: writeconcern
    value: <REPLACE-WITH-WRITE-CONCERN> # Optional.
  - name: readconcern
    value: <REPLACE-WITH-READ-CONCERN> # Optional.
  - name: operationTimeout
    value: <REPLACE-WITH-OPERATION-TIMEOUT> # Optional.
  - name: params
    value: <REPLACE-WITH-ADDITIONAL-PARAMETERS> # Optional.
```

For example the operationTimeout can probably be combined with the network interruption somehow.

Reply:
Fixed